pkgs/development/ocaml-modules/janestreet/janePackage_0_15.nix

```diff
@@ -9,7 +9,7 @@
 , ...}@args:
 
 buildDunePackage (args // {
-  useDune2 = true;
+  duneVersion = "2";
```

Comment:
This cause failure because args defined `duneVersion` and the resulting set contained the two (`useDune2` and `duneVersion`). `duneVersion` is read first so it was un-noticed. With this change we get two definition of `duneVersion` and the value is always overwritten. RETEX: when using this pattern to provide default argument write: { default } // args.

With the requested change we remove the double `duneVersion` so changing might be un-needed.